src/styles/ia-book-search-results.js

```diff
@@ -14,8 +14,8 @@ export default css`
 
 header {
   display: flex;
-  justify-content: space-between;
   align-items: center;
+  padding: 0 calc(var(--closeMenuIconWidth) + 1.5rem) 2rem 0;
```

Comment:
```suggestion
padding: 0 2rem;
```

Reply:
If we're going to remove the header and extra actions from each of these components, there's no need for this padding value. If we're keeping the header in each submenu, I believe the given spacing from Jim was 27px, or 2.7rem, on the right side of the header. This suggested change is also removing the bottom padding on the header, which would put it in direct contact with the panel's content.

Reply:
ok thanks. will keep in mind as we continue forward.